bin/roave-infection-static-analysis-plugin

```diff
@@ -59,7 +60,10 @@ use function var_export;
     $makeAnalyzer = static function () use ($projectPath): ProjectAnalyzer {
         RuntimeCaches::clearAll();
 
-        $config = Config::getConfigForPath($projectPath, $projectPath);
+        $configDirectory = getenv('PROJECT_CONFIG_DIRECTORY') ?: '';
+        $configPath = '' !== $configDirectory ? ($projectPath . '/' . $configDirectory) : $projectPath;
```

Comment:
This is what the `-c` flag does for psalm, no? Is it not being respected at all?

The problem here is that we obviously have flags from both infection and psalm.

IMO, preserving the `psalm.xml` or `psalm.xml.dist` at the root of a software project is simpler than introducing more customization to this project to make ends meet elsewhere.